webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue

```diff
@@ -23,49 +27,27 @@
         </option>
       </select>
     </v-scale-transition>
+    <v-scale-transition >
+      <v-btn
+          v-show="isMobile && showMobileFilter || !isMobile"
+          class="btn px-2 btn-primary"
+          outlined
+          @click="openPeopleAdvancedFilterDrawer()"
+      >
+        <i class="uiIcon uiIcon24x24 settingsIcon primary--text mr-1"></i>
```

Comment:
POs had deprecated the use of PLF-Fonts & Material Design icons, please use Font-Awesome icons. cc @srenault-meeds can help if needed.

```suggestion
        <v-icon color="primary">fa-...</v-icon>
```

Reply:
fixed